### ISSUE TITLE
Zoom feature - drill into a subnode

### ISSUE THREAD

**Issue by NickEmpetvee** [frontend-collective/react-sortable-tree#468]
My use case is that I often expand nodes and focus on subnode deep into the tree. When you have a broad and deep tree, display is challenging within the confines of a portlet.

Is there a way to display specify where in the whole treeData to start the top-level display? For example if a hierarchy within treeData is `a - b - c - ...n`, can you zoom in to only show from 'b' onwards, or just from 'b' to 'c'?

**Comment by gragland** [comment]
I think you’d need to implement this yourself outside of sortable tree by passing it the portion of the tree that you’ve zoomed to.

**Comment by NickEmpetvee** [comment]
@gragland yeah most likely. I was just seeing if something existed that I could leverage.
